modules/adding-ibm-z-lpar-agent.adoc

```diff
@@ -35,6 +35,19 @@ random.trust_cpu=on rd.luks.options=discard
 <1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are starting. Only HTTP and HTTPS protocols are supported.
 <2> For the `ip` parameter, manually assign the IP address, as described in _Installing a cluster with z/VM on IBM Z and IBM LinuxONE_.
 <3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where {op-system-first} is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where {op-system} is to be installed.
++
+[NOTE]
+====
+For FCP multipath configurations, provide two disks instead of one.
+====
++
+.Example 
+[source,yaml]
+----
+rd.zfcp=<adapter1>,<wwpn1>,<lun1> \
+rd.zfcp=<adapter2>,<wwpn2>,<lun2>
+----
```

Comment:
If you look in the preview of this PR, the addition of this note and code block breaks the formatting/order of callouts that follow this. You may want to try something like wrapping this content in an open delimiter block (see [this page](https://docs.asciidoctor.org/asciidoc/latest/blocks/delimited/#:~:text=....-,open,-%3Aopen) for more details), which will sort of put all of this content in a separate "block" of asciidoc text.

```suggestion
--
[NOTE]
====
For FCP multipath configurations, provide two disks instead of one.
====
+
.Example 
[source,yaml]
----
rd.zfcp=<adapter1>,<wwpn1>,<lun1> \
rd.zfcp=<adapter2>,<wwpn2>,<lun2>
----
--
```